### PR TITLE
HC-914 Uninstall Arduino IDE

### PR DESCRIPTION
**Pull request by seaxwi**
**Changes proposed in this pull request**
Content and structure improvements.

### CODE REVIEW

File: content/Software and Downloads/Installation/Uninstall-Arduino-IDE.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+[The Sketchbook](https://support.arduino.cc/hc/en-us/articles/4412950938514) contains your saved sketches and installed libraries. It is located in your Documents folder by default, inside the `Arduino` subfolder.
+
+* **Windows:** `C:\Users\{username}\Documents\Arduino`
+* **macOS:** `/Users/{username}/Documents/Arduino`
+* **Linux:** `/home/{username}/Arduino`
+
+[The Arduino15 folder](https://support.arduino.cc/hc/en-us/articles/360018448279). Located in these locations by default:
+
+* **Windows:** `C:\Users\{username}\AppData\Local\Arduino15`. Open your Users older in Explorer and look for a folder called `Appdata`. This folder is hidden by default. In newer versions of Windows, you can display hidden items by clicking the View tab, and checking Hidden items.
```
Comment (per1234) [inline content/Software and Downloads/Installation/Uninstall-Arduino-IDE.md:34]:
```suggestion
* **Windows:** `C:\Users\{username}\AppData\Local\Arduino15`. Open your Users folder in Explorer and look for a folder called `AppData`.
```
- Fix typos
- Remove sentence that is a duplicate of the sub-bullet point

File: content/Software and Downloads/Installation/Uninstall-Arduino-IDE.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  * This folder is hidden by default. I newer versions of Windows, you can display hidden items by clicking the **View** tab, and checking _Hidden items_.
```
Comment (per1234) [inline content/Software and Downloads/Installation/Uninstall-Arduino-IDE.md:35]:
```suggestion
  * This folder is hidden by default. In newer versions of Windows, you can display hidden items by clicking the **View** tab, and checking _Hidden items_.
```

**Review by per1234 — CHANGES_REQUESTED** [review]

**Review by Renat0Ribeir0 — APPROVED** [review]

**Review by Renat0Ribeir0 — APPROVED** [review]
Besides the suggestions from @per1234 LGTM 👍

File: content/Software and Downloads/Installation/Uninstall-Arduino-IDE.md (hunk 1 above)
Comment (seaxwi) [reply content/Software and Downloads/Installation/Uninstall-Arduino-IDE.md:34]: Thanks

**Review by per1234 — APPROVED** [review]

**Pull request merged**